### PR TITLE
Kubernetes setup with README explanation

### PR DESCRIPTION
**Pull request by mr-rooftop**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,18 +5,42 @@ If you have questions feel free to open a github issue or contact us in our Tele
...
+
+Install the same `kubectl` version you use in your K8s cluster (one major version difference is alright).
+
+Download the cluster config file from your K8s provider, and set the path of it to the `KUBECONFIG` environment 
+variable:
+```
+export KUBECONFIG=/your/path/to/the/moon/k8s-kubeconfig.yaml
+```
+
+Now run
```
Comment (mbartn) [inline README.md:36]:
```suggestion
Now, from the project's main directoty, run 
```

File: README.md [hunk 2]
```diff
@@ -132,8 +157,71 @@ If you want to reset your bot's data, simply delete the file `session.data` in t
...
 To get real data just set `DEBUG=False` and all other environment variables as 
 described in the 'Set environment variables' section.
-If you're using docker-compose to run this Bot, modify the existing variables in `variables.env` file (No need to
-set DEBUG as there's no DEBUG mode in the docker version). 
+
+If you're using kubernetes to run this Bot, modify the existing variables in `kubernetes/k8s_thornode_bot_deployment.yaml` file.
+To use docker-compose, modify the existing variables in `variables.env` file (No need to
+set DEBUG as there's no DEBUG mode in the k8s or docker version). 
+
+### [Kubernetes](#kubernetes)
```
Comment (mbartn) [inline README.md:165]: What's the difference between this section and 
```
### Kubernetes (K8s)
```
from line 24?

How about merging the previous section with this one?

Comment (mbartn) [inline README.md:165]:
```suggestion
### [Kubernetes](#kubernetes)
#### Setup
```
How do you see divding the whole kubernetes section into:
- Setup
- Modyfing existing deployment (?)

More in next comments

File: README.md [hunk 3]
```diff
@@ -132,8 +157,71 @@ If you want to reset your bot's data, simply delete the file `session.data` in t
...
+```
+
+Check out if your deployment and persistent volume claim were succesfully created with:
+```
+kubectl get pods -n thornode-bot
+kubectl get pvc -n thornode-bot
+``` 
+
+---
+
```
Comment (mbartn) [inline README.md:204]:
```suggestion
#### Modifying the running deployment (?)
```

File: README.md [hunk 4]
```diff
@@ -132,8 +157,71 @@ If you want to reset your bot's data, simply delete the file `session.data` in t
...
+Then delete the deployment, and create it again:
+```
+kubectl create -f kubernetes/k8s_thornode_bot_deployment.yaml
+kubectl delete -f kubernetes/k8s_thornode_bot_deployment.yaml
+```
+
+Your bot should send you a message that he got updated. 
+He now uses the new environment variables / newest docker image.
+
+---
```
Comment (mbartn) [inline README.md:217]:
```suggestion
```

Comment (mbartn) [reply README.md:217]: (if you decide to divide k8s section)

**Review by mbartn — CHANGES_REQUESTED** [review]
I ran kubernetes with this docs and everything went smootly even though i don't have any experience with k8s.

I haven't check logical part of the `.yaml` files as I would need to spend a few hours for actually reading about k8s stuff. If you think that's necessary just let me know.

File: README.md (hunk 2 above)
Comment (mr-rooftop) [reply README.md:165]: The first section is part of the "quickstart" section, which aims to give minimal information to set everything up.
The latter section explains the K8s setup in more detail.
IMO it makes sense to provide the user with the easy to use "quickstart" section in addition to the lengthier detail sections.

Comment (mr-rooftop) [reply README.md:165]: Neat idea

Comment (mbartn) [reply README.md:168]: ok, so let's leave that

**Review by mbartn — APPROVED** [review]

**Pull request merged**